### DISCUSSION TITLE
unordered_map

### DISCUSSION THREAD

**Discussion by mikesreed** [nlohmann/json#2801] [Q&A] [answered]
Hi,
I tried to instantiate basic_json specifying the ObjectType as std::unordered_map but it failed to compile.  Is it intended to support std::unordered_map as the ObjectType?  I wanted to see if, in my circumstances, the performance was better than the default std::map.
I was using gcc 8,2,1.
If this is meant to be supported I can give more details of the error (usual reams of template diag)
Mike.

**Comment by nlohmann** [comment, marked as answer]
No, it’s currently not supported.
